pkg/src/src/main/scala/edu/berkeley/cs/amplab/sparkr/SparkRBackendHandler.scala

```diff
@@ -144,6 +144,7 @@ class SparkRBackendHandler(server: SparkRBackend) extends SimpleChannelInboundHa
       args: Array[java.lang.Object],
       parameterTypes: Array[Class[_]]): Boolean = {
     if (parameterTypes.length != numArgs) {
+      // println(s"num of arguments numArgs not match with ${parameterTypes.length}")
```

Comment:
Ah I see - okay. I didn't realize this gets called for every method. In that case lets delete this line. The we should move the  println line to the top method to when `methods.isEmpty` is true to print `cannot find matching method $methodName. Candidates are $printMethods(methods)` or something like that. Does that sound better ?